### PR TITLE
Define basic content for Readme file

### PR DESCRIPTION
**Pull request by mateuszgorniak**
Just define basic content for Readme about setup, basic configuration, running up application and testing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,90 @@
...
+### :unlock: Access
+
+| Environment | Address |
+| :--: | :---: |
+| Production | codestories-review-bot.herokuapp.com |
+
+### :new: Deployments
+
+To make deployments and manage the apps you need privileges to the Heroku's project.
+Automatically deployments to the Production are already enabled for `master` branch.
```
Comment (mszpakowski) [inline README.md]: Automatic :)

File: README.md [hunk 2]
```diff
@@ -1,19 +1,90 @@
...
+### :pencil: Default configuration
+
+Define local values for the environment variables defined in `config\dev.exs` file.
+
+### :rocket: Running server on the local machine
+
+Then configure your database in config/dev.exs and run:
+
+```
+$ mix ecto.create
```
Comment (mszpakowski) [inline README.md:50]: `mix ecto.migrate` also

File: README.md [hunk 3]
```diff
@@ -1,19 +1,90 @@
...
+* Install all dependencies used by the Project
+
+```
+$ cd codestories-review-bot
+$ mix deps.get
+```
+
+### :pencil: Default configuration
+
+Define local values for the environment variables defined in `config\dev.exs` file.
```
Comment (mszpakowski) [inline README.md]: `config/dev.exs` 😉

File: README.md [hunk 4]
```diff
@@ -1,19 +1,91 @@
...
+
+### :rocket: Running server on the local machine
+
+Then configure your database in `config\dev.exs` and run:
```
Comment (mszpakowski) [inline README.md]: You changed the other one, but at least it's consistent now 😄

**Review by mszpakowski — APPROVED** [review]

File: README.md (hunk 4 above)
Comment (mateuszgorniak) [reply README.md]: Hahah sorry, It was too late to think xD Fixed 👍

**Pull request merged**